tests/core/stdlib/test_reproduce.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Our CI won't be able to run this test. 

I think it would be better to have a narrower test which depends on `@io_bazel_rules_go//:stdlib`, and verifies that some file (say `fmt.a`) does not have a build id using `go tool buildid`.

This is not a straightforward test, so feel free to leave it as a TODO for me.

Reply:
That is a very good idea! Although I think we should keep the `.sh` to ensure that `stdlib` is reproducible, maybe by hand sometimes?